source/beamline/VESPERS/VESPERS13ElementGeDetector.h

```diff
@@ -0,0 +1,71 @@
+#ifndef VESPERS13ELEMENTGEDETECTOR_H
+#define VESPERS13ELEMENTGEDETECTOR_H
+
+#include "beamline/AMXRFDetector.h"
+
+/// Implementation of the AMXRFDetector for the 13 element germanium detector on VESPERS (shared with IDEAS).
+class VESPERS13ElementGeDetector : public AMXRFDetector
+{
+	Q_OBJECT
+
+public:
+	/// Default constructor.  Requires the name and description of the detector.  It builds all the necessary components.
+	VESPERS13ElementGeDetector(const QString &name, const QString &description, QObject *parent = 0);
+	/// Destructor.
+	virtual ~VESPERS13ElementGeDetector();
+
+	/// The germanium detector doesn't explicitly require powering on
+	virtual bool requiresPower() const { return false; }
+
+	/// Cancelling is implemented for the germanium detectors
+	virtual bool canCancel() const { return true; }
+	/// Clearing is not currently supported for the germanium detectors
+	virtual bool canClear() const { return false; }
+
+	/// Germanium detectors are not currently capable of continuous acquisition
+	virtual bool canContinuousAcquire() const { return false; }
+
+	/// The germanium can be configured to work with synchronized dwell time systems
+	virtual bool supportsSynchronizedDwell() const { return false; }
```

Comment:
The difference between the comment here and the return result regarding whether is supports sync-dwell seems confusing.